scripts/build-json/build-page-json.js

```diff
@@ -47,21 +39,26 @@ async function buildPageJSON(elementRelativePath) {
     const prosePath = path.join(elementPath, 'prose.md');
     const contributorsPath = path.join(elementPath, 'contributors.md');
 
-    // make the package
-    const element = {};
-    element.title = meta.title;
-    element.mdn_url = meta['mdn-url'];
-    element.interactive_example_url = meta['interactive-example'];
-    element.browser_compatibility = bcd.package(meta['browser-compatibility']);
+    // set up element data
+    const element = {
```

Comment:
Does this variable name make sense any more?

Reply:
No, but this whole thing will need redoing at some point to be not just about HTML elements, and I don't think this is the point for that.